safe/impact_functions/earthquake/itb_earthquake_fatality_model/impact_function.py

```diff
@@ -126,20 +126,52 @@ def __init__(self):
         ])
         self.total_fatalities = None
 
-    def fatality_rate(self, mmi):
+    def compute_fatality_rate(self):
         """ITB method to compute fatality rate.
 
```

Comment:
.. versionchanged:: 3.3 - renamed method from fatality_rate to compute_fatality_rate